src/atoms/Spacer/Readme.md

```diff
@@ -0,0 +1,44 @@
+Extra Small:
+
+    <div>
+    <div style={{ backgroundColor: '#b3b3b3', width: '100%', height: '100px'}}></div>
```

Comment:
@jmcolella Consider creating a one-off example component and just giving it a dotted outline?